package.json

```diff
@@ -42,7 +42,7 @@
   },
   "dependencies": {
     "class-is": "^1.1.0",
-    "libp2p-crypto": "^0.21.0",
+    "libp2p-crypto": "^0.22.8",
```

Comment:
I don't see a 0.22.x released yet

Reply:
It's there: https://www.npmjs.com/package/@libp2p/crypto

Reply:
`@libp2p/crypto` is ESM only and will break things in this module as it cannot be `require`d.